v2/app/collegevine-hub/CVHubViewer.jsx

```diff
@@ -1751,6 +1751,10 @@ var PivotViewer = (Pivot.PivotViewer = function(
         viewport.goHome()
       }
 
+      if (selectedItem) {
+        self.trigger("itemClicked", selectedItem.id)
```

Comment:
Could we do the following…?

```suggestion
        self.trigger("itemClick", selectedItem)
```

- Present tense: I just realized Pivot is not very consistent, e.g. `finishedRearrange`, `itemsCleared`, `hideDetails`, etc., but the HTML spec is pretty clear: `click`, `mousemove`, etc.
- Let’s keep `camelCase`. This aligns with Pivot itself (exceptions: `animationstart` + `animationfinish`) and modern APIs, e.g. React `onClick`, etc.
- Let’s return the entire `selectedItem`. If we just return a string instead of an object, it will become backwards incompatible if we ever want to return more properties. Therefore, most libraries return objects from event handlers, e.g. `{ id :: String }` instead of jus `String`, so new properties can be added without making it a breaking change.

Reply:
No worries will do :+1: